easybuild/easyconfigs/t/twine/twine-1.13.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'PythonBundle'
+
+name = 'twine'
+version = '1.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://twine.readthedocs.io'
+description = "Twine is a utility for publishing Python packages on PyPI."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [('Python', '3.6.6')]
+
+#use_pip = True
```

Comment:
Travis complains on this!

Reply:
There was a reason I had to use this, but I don't recall what it was...